engines/net.c

```diff
@@ -223,6 +235,16 @@ static inline int is_ipv6(struct netio_options *o)
 	return o->proto == FIO_TYPE_UDP_V6 || o->proto == FIO_TYPE_TCP_V6;
 }
 
+static inline int is_vsock_stream(struct netio_options *o)
```

Comment:
You use this one without the ifdef protection, yet is_vsock() is. Get rid of one of these helpers as they do the same thing, and get rid of the ifdef around calling them.

Reply:
As of now the two helpers to the same thing, yes.
The reason behind having both of them is that in the future someone could add support for VSOCK_DGRAM too, possibly with a `is_vsock_dgram()` helper.

The body of `is_vsock()` would simply change into

```c
return is_vsock_stream(o) | is_vsock_dgram(o);
```

Reply:
Then in that future, someone could add separate helpers. For now it doesn't make sense, don't over design.